src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -184,6 +185,14 @@ func (a *App) onBootstrapWorkerNode(ctx context.Context, s state.State, encodedT
 		localhostAddress = "127.0.0.1"
 	}
 
+	var securePort = "6443"
```

Comment:
super small nit: move this below the apiserver check. Otherwise this variable is set without usage.
Also, don't use `var` for initialization. `securePort := "6443"` is more ideomatic.